e2e-tests/non-voting/run

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+test_dir=$(realpath $(dirname $0))
+. ${test_dir}/../functions
+
+check_cr_config() {
+	local cluster="$1"
+
+	# apply cr config
+	apply_cluster $test_dir/conf/$cluster.yml
+
+	# wait for readiness
+	wait_for_running $cluster 2 "true"
+
+	# check if statefulset created with expected config
+	compare_kubectl statefulset/$cluster-nv
+
+	# delete cluster
+	kubectl_bin delete \
```

Comment:
not needed, since `destroy` will also delete cluster

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/723/commits/78b976cdc932c4381c8343c0811c45e8feddb755